<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calendar</class>
 <widget class="QDialog" name="calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#calendar{background-image: url(:/new/myimage/new_icon/Background.jpg);}</string>
  </property>
  <widget class="QPushButton" name="goback">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>600</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{	
	color:#000000;
	border:2px solid #ffffff;
	background-color:#bbcbd8;
	border-radius:5px;
}

QPushButton::hover{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::pressed,QPushButton::checked{
	background-color:#52DCFE;
	color:#000000;
}

QPushButton::disabled{
	background-color:transparent;
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>461</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 203, 216);
font: 16pt &quot;华文楷体&quot;;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_date">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*双击某一日期可查看/修改该日账目细则～</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>该日收入：</string>
   </property>
  </widget>
  <widget class="QLabel" name="income_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>   * * *</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>该日支出：</string>
   </property>
  </widget>
  <widget class="QLabel" name="outcome_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>560</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>   * * *</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>580</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
